src/index.js

```diff
@@ -0,0 +1,41 @@
+import './index.css';
+
+const items = [
+  {
+    description: 'wash dishes',
+    completed: true,
+    index: 3,
+  },
+  {
+    description: 'Laundary',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'cleaning the house',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'shopping',
+    completed: true,
+    index: 2,
+  },
+];
+
+const sortedItems = items.sort((a, b) => a.index - b.index);
+const wrapper = document.getElementById('task-wrapper');
+sortedItems.forEach((item) => {
+  const task = document.createElement('li');
+  const desc = document.createElement('p');
+  desc.classList = 'task-desc';
```

Comment:
## [OPTIONAL] i mark this as optional because the previous reviewer missed this requirement 
* I strongly sugesst you implement it, This requirement : `On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.` this means on page load todo with the first index should be rendered first and followed accordingly [This is the link to the requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=On%20page%20load%20render%20the%20dynamically%20created%20list%20of%20tasks%20in%20the%20dedicated%20placeholder.%20The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.)

Reply:
Hi David, 

Its already rendering according to the index values. Could you please specify what exactly referring to?